apps/govern/components/Contracts/EditVotes/index.tsx

```diff
@@ -74,7 +74,7 @@ const getColumns = (
           value={allocations[index].weight}
           status={isError ? 'error' : undefined}
           onChange={(value) => {
-            if (value) {
+            if (typeof value === 'number') {
```

Comment:
this fixes the ability to reset existing votes to zero @kupermind you complained about that it returns the value to the previous one